neqo-transport/src/server.rs

```diff
@@ -167,18 +166,8 @@ pub struct Server {
     cid_generator: Rc<RefCell<dyn ConnectionIdGenerator>>,
     /// Connection parameters.
     conn_params: ConnectionParameters,
-    /// Active connection attempts, keyed by `AttemptKey`.  Initial packets with
-    /// the same key are routed to the connection that was first accepted.
-    /// This is cleared out when the connection is closed or established.
-    active_attempts: HashMap<AttemptKey, StateRef>,
     /// All connections, keyed by `ConnectionId`.
     connections: ConnectionTableRef,
-    /// The connections that have new events.
-    active: HashSet<ActiveConnectionRef>,
```

Comment:
Instead of keeping track which `Connection` has new events, ask the `Connection`s ad-hoc whenever the information is needed.